tests/backup/backup_locked_bucket_test.go

```diff
@@ -134,8 +135,11 @@ var _ = Describe("{BackupAlternatingBetweenLockedAndUnlockedBuckets}", func() {
 			log.FailOnError(err, "Fetching px-central-admin ctx")
 			err = CreateSourceAndDestClusters(orgID, "", "", ctx)
 			dash.VerifyFatal(err, nil, "Creating source and destination cluster")
-			clusterStatus, clusterUid = Inst().Backup.RegisterBackupCluster(orgID, SourceClusterName, "")
-			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying backup cluster %s registration", SourceClusterName))
+			clusterStatus, err = Inst().Backup.GetClusterStatus(orgID, SourceClusterName, ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching [%s] cluster status", SourceClusterName))
```

Comment:
Can we use log.FailOnError instead of dash.VerifyFatal for this verification to avoid too many log messages as now all the test cases will have this
[Not a mandatory change, only if it makes sense]

Reply:
I am currently using dash.VerifyFatal because it was used before. Perhaps, we can revisit this once we establish a standardized structure and syntax for the test case.